docs/rfc/rfc-1.adoc

```diff
@@ -545,6 +572,8 @@ Alphabetized list of Governable Parameters with additional notes.
   <<heartbeat,heartbeat>> to be successful.
 - `heartbeat_block_length`: The number of ethereum blocks until the next heartbeat.
   If set to 40, then the signers sign every 40th block.
+- `skip_sweep_timeout`: The amount of time the depositor has to reimburse the
+  operator for the gas of the sweep and collect their account balance.
```

Comment:
My only worry is:
- The time starts counting after the deposit is swept and that the time it takes to sweep is unknown - hard UX; this timeout needs to be long.
- We probably need some minimum BTC deposit amount (for standard sweep as well!) so that the cost of sweep can not be higher than the fee from BTC amount deposited. This is a bigger issue to think about so not blocking here.